docker/docker-compose-single-dc-setup.yaml

```diff
@@ -76,3 +76,17 @@ services:
     depends_on:
       venice-router:
         condition: service_healthy
+
+  venice-client-jupyter:
+    build:
+      args:
+        spark_version: 3.1.2
+    image: venicedb/venice-client-jupyter:latest-dev
```

Comment:
The main issue with using latest-dev is that it will disrupt this flow: [Quickstart - Single Datacenter](https://venicedb.org/docs/quickstart/quickstart-single-datacenter).

Would it be possible to create Docker images tagged with `0.4.336` (based on code at that tag), push them, and then use this tag consistently for all Venice images?

Alternatively we can just push  `venicedb/venice-client-jupyter:latest-dev` to dockerhub for now

Reply:
Update: I've pushed docker images with tag `0.4.340` to [docker hub](https://hub.docker.com/u/venicedb). Could you please replace tags for `venice-` images with `0.4.340`? Thanks